pkg/operator/controller/status_test.go

```diff
@@ -434,3 +437,303 @@ func TestOperatorStatusesEqual(t *testing.T) {
 		}
 	}
 }
+
+func TestComputeOperatorStatusVersions(t *testing.T) {
```

Comment:
Consider a more compact table layout, e.g.

```go
{
  description: "update operator and ingress controller image, operator is available and progressing",
  inputs: testInputs{
    oldReleaseVersion:         "v1",
    oldIngressControllerImage: "ic-v1",
    curReleaseVersion:         "v2",
    curIngressControllerImage: "ic-v2",
    availableCondition:        true,
    progressingCondition:      true,
  },
  outputs: testOutputs{
    releaseVersion:         "v1",
    ingressControllerImage: "ic-v1",
    versionChanged:         true,
    ingressImageChanged:    true,
  },
},
```

becomes:

```go
{
  desc: "update operator and ingress controller image, operator is available and progressing",
  start: input{fromVersion:"v1", fromImage: "ic-v1", toVersion: "v2", toImage: "ic-v2", available: true, progressing: true},
  expect: output{version: "v1", image: "ic-v1", versionChanged: true, imageChanged: true}
},
```

Just for example

Reply:
Or perhaps:

```go
{
  desc: "update operator and ingress controller image, operator is available and progressing",
  from: versions{operator:"v1", operand: "ic-v1"},
  to: versions{operator:"v2", operand: "ic-v2"},
  in: condition{available: true, progressing: true},
  expect: expect{
    versions: versions{operator: "v2", operand: "ic-v2"},
    in: condition{versionChanged: true, imageChanged: true},
  },
},
```

Reply:
Changed the test table to compact layout.